docs/sources/k6/next/javascript-api/k6-browser/locator/evaluate.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'evaluate(pageFunction[, arg])'
+description: 'Browser module: locator.evaluate(pageFunction[, arg]) method'
+---
+
+# evaluate(pageFunction[, arg])
+
+Executes JavaScript code in the page, passing the matching element of the locator as the first argument to the `pageFunction` and arg as following argument(s). It returns the value of the `pageFunction` invocation.
+
+<TableWithNestedRows>
+
+| Parameter    | Type               | Defaults | Description                                  |
+| ------------ | ------------------ | -------- | -------------------------------------------- |
+| pageFunction | function or string |          | Function to be evaluated in the page context.                    |
+| arg          | string             | `''`     | Optional argument to pass to `pageFunction`. |
+
+</TableWithNestedRows>
```

Comment:
```suggestion

| Parameter    | Type               | Defaults | Description                                  |
| ------------ | ------------------ | -------- | -------------------------------------------- |
| pageFunction | function or string |          | Function to be evaluated in the page context.                    |
| arg          | string             | `''`     | Optional argument to pass to `pageFunction`. |

```

We don't need this HTML tag, I think it's leftover from the previous docs. I'm gonna open a separate PR to remove this from all the other pages. 🤓